logdetective/server.py

```diff
@@ -1,21 +1,29 @@
 import logging
 import os
 import json
-
+from llama_cpp import CreateCompletionResponse
 from fastapi import FastAPI, HTTPException
 from pydantic import BaseModel
 
 import requests
 
 from logdetective.constants import PROMPT_TEMPLATE
 from logdetective.extractors import DrainExtractor
-from logdetective.utils import validate_url
+from logdetective.utils import validate_url, compute_certainty
 
 class BuildLog(BaseModel):
     """Model of data submitted to API.
     """
     url: str
 
+
+class Response(BaseModel):
+    """Model of data returned by Log Detective API
+    """
+    llm_response: CreateCompletionResponse
```

Comment:
Can we see an example content of this?

Reply:
Sure, it's all part of llama-cpp-python API https://llama-cpp-python.readthedocs.io/en/latest/api-reference/#llama_cpp.llama_types.CreateCompletionResponse

Reply:
nice, that's pretty detailed

please link it in the code so it's easy to find :)

Reply:
Done